website/static/css/fangorn.css

```diff
@@ -29,6 +29,9 @@ text-align: center;
 .fangorn-selected .text-muted {
     color: #eeeeee;
 }
+.fangorn-selected .text-danger {
+    color: #FFC6C5;
```

Comment:
This color could use improvement. The text is still hard to read when the row is selected.

![cursor_and_osf___addons_example_files](https://cloud.githubusercontent.com/assets/2379650/8853294/fcac95b8-3127-11e5-86e2-1b45997a7f93.png)

Perhaps just use white?